### ISSUE TITLE
Cmake modernization

### ISSUE THREAD

**Issue by sobolevnrm** [Electrostatics/apbs-pdb2pqr#550]

**Comment by sobolevnrm** [comment]
Move entire code to Cmake build system.

**Comment by intendo** [comment]
This has been mostly completed for APBS except for the fetk submodule that uses CMakes ExternalProject_Add conventions to build the fetk sections.

At this point, I need to know how to get pdb2pqr to build with CMake before being able to close this out.

Is @sobolevnrm close to merging pdb2pqr into the master branch?

**Comment by sobolevnrm** [comment]
Is @intendo giving me a hard time? 🤔 

It depends what you want me to merge.

I have been spending most of my time on getting PROPKA ready for re-integration with PDB2PQR.  As soon as pull requests https://github.com/jensengroup/propka-3.1/pull/50 and https://github.com/jensengroup/propka-3.1/pull/59 are approved, we'll have a PROPKA version that I think we can call v3.2 and import as a module.

In the meantime, I've been working on the PDB2PQR interface with PROPKA to see what aspects of PROPKA ligand support could be used in the main part of PDB2PQR.

I have not done anything with PDB2PKA.  The PDB2PQR/PDB2PKA interface is confusing and I still haven't figured out what's going on internally within PDB2PKA for ligand support.  PDB2PKA could be another priority (rather than all of PDB2PQR) for the CRC analysis you're planning.

In summary, you can merge a PDB2PQR right now that has no pKa support, wait a week or so for PROPKA support, or wait a long time for all functionality to be restored.

**Comment by intendo** [comment]
Hard time? I am just trying to catch up to you  :-)

I think we can wait a week or two to get PROPKA working.
